src/borg/archiver/help_cmd.py

```diff
@@ -180,6 +178,25 @@ class HelpMixIn:
         exclude stops examination of subdirectories so that potential includes
         will not match - use normal excludes for such use cases.
 
+        Example::
+
+            # Define the root path prefix
+            R /
+            # Exclude all iso files in any directory
+            - **/*.iso
+            # Explicitly include some directories and all their content
+            + etc/**
+            + root/**
```

Comment:
maybe this does **not** back up the `etc` directory item?

Reply:
also, this is a bit unusual.

many users would consider these 2 dirs as important and backup worthy rather than wanting to exclude them.

Reply:
```
root@osz ~# borgmatic mount --repository vps --archive osz-2023-06-08T20:29:47 --mount-point /mnt/m
root@osz ~# cd /mnt/m
root@osz /m/m# ls
etc/  home/  root/
```

> backup worthy rather than wanting to exclude them

I don't understand why you think that it could not backup the /etc dir ?

? I *include* them, not exclude, so I don't get your second comment.

Reply:
ehrm, ignore my second comment, guess i was distracted or it was too late already.

maybe "include all inside etc and root" or so would be a good comment.
the 2 directory fs items might be missing, but as they are already present after a fresh install, this is no real issue.